### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by sbercx**
Made it work with current versions of the dependencies. Also applied rustfmt. Works with stable 1.14 and current nightly.

### CODE REVIEW

**Comment by elifoster** [comment]
Can you resubmit this (or undo and just push again) without the formatting changes?

**Review by elifoster — CHANGES_REQUESTED** [review]
Can you resubmit this (or undo and just push again) without the formatting changes?

**Comment by sbercx** [comment]
I've reverted the formatting.

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "cleverbot_io"
-version = "1.0.0"
+version = "1.0.1"
```
Comment (elifoster) [inline Cargo.toml]: Can you undo this version bump?

**Review by elifoster — CHANGES_REQUESTED** [review]

**Comment by elifoster** [comment]
Looks good except for that one little change I requested. I'll test it by the weekend. Thanks for the contribution :smile:

**Comment by sbercx** [comment]
There you go!

**Review by elifoster — APPROVED** [review]

**Pull request merged**